### ISSUE TITLE
MultiQC Metadata

### ISSUE THREAD

**Issue by TMiguelT** [ewels/MegaQC#92]
Currently the report parsing function will *only* import keys in the MultiQC report that begin with `config_`. However, I can envisage there are cases where users will want to add other metadata to the report, that should be queryable in MegaQC.

I'm no expert in MultiQC, but are `config_` fields configurable? If not, is there some other mechanism MultiQC provides for adding arbitrary metadata? Note that I'm not asking about *presentation* information, which the MultiQC documentation is mostly concerned with. I'm talking about things like "pipeline_version", which might be a string that defines the version of the analysis pipeline that generated this report. Where would that go in a MultiQC report?

**Comment by ewels** [comment]
Yup, I have planned this - should be an issue about it somewhere, but can’t find it currently. It will be part of the MultiQC “custom content” support, so look like it came from a module. Only difference is that it needs an option to not be present in the MultiQC report. Also would be nice to have an option for dl lists instead of just tables (this is how I usually do software versions, but as a HTML string which doesn’t work well with MegaQC).
